nixos/modules/services/web-apps/immich.md

```diff
@@ -0,0 +1,265 @@
+# Immich {#module-immich}
+
+*Source:* {file}`modules/services/web-apps/immich.nix`
+
+*Upstream documentation:* <https://immich.app/docs/overview/introduction/>
+
+[Immich](https://immich.app/) is a self-hosted photo and video management solution.
+
+## Configuring {#module-immich-configuring}
+
+A complete list of options for the Immich module may be found
+[here](#opt-services.immich.enable).
+
+### Basic setup {#module-immich-configuring-basic-setup}
```

Comment:
Overall nit: make the subtitles more consistent in their capitalization :)

Reply:
They should be more consistent now. At the end I chose not to go with title case, as I felt it might be out of place.